src/Contacts/Contact.h

```diff
@@ -1,8 +1,23 @@
+//
+//  Copyright (c) 2017 Open Whisper Systems. All rights reserved.
+//
+
 #import <AddressBook/AddressBook.h>
 #import <Foundation/Foundation.h>
 
 NS_ASSUME_NONNULL_BEGIN
 
+typedef NS_ENUM(NSUInteger, OWSPhoneNumberType) {
+    OWSPhoneNumberTypeUnknown,
+    OWSPhoneNumberTypeMobile,
+    OWSPhoneNumberTypeIPhone,
+    OWSPhoneNumberTypeMain,
+    OWSPhoneNumberTypeHomeFAX,
```

Comment:
I take it back, I guess you were just following the ABConvention: `kABPersonPhoneHomeFAXLabel`